Resources/Maps/_NF/Shuttles/stasis.yml

```diff
@@ -3201,6 +3197,8 @@ entities:
       pos: 11.5,-1.5
       parent: 1
     - type: DeviceNetwork
+      configurators:
+      - invalid
```

Comment:
Remove any instances of this you see - they result in "Error: invalid entity with UID '0'" errors when the ships are purchased.  Has to be done after saving, and not sure why it happens.  Only labeling the one, but do remove _all instances_.
```suggestion
```